src/components/blog/RelatedArticles.js

```diff
@@ -51,7 +52,7 @@ const RelatedArticles = ({relatedPosts}) => {
         <RelatedArticlesHeading as="h6">Related articles:</RelatedArticlesHeading>
         <RelatedArticlesContainer>
           {relatedPosts.map(({ slug, frontmatter, excerpt }) => (
-            <BlogListItem
+            <PostShort
```

Comment:
I think we can remove `BlogListItem.js`  file since it is not being used anywhere.

Reply:
already removed 😃